src/utils/Tmdb.js

```diff
@@ -0,0 +1,180 @@
+   * @param {string} tvId
+   * @param {string} seasonNumber
+   * @param {string} episodeNumber
+   * @param {Array<string>} [subPath]
+   * @param {Object} [searchParams] - see query() for details
+   * @returns {Promise<Response>} - see query() for details
+   */
+  getTvEpisode(tvId, seasonNumber, episodeNumber, subPath = [],  searchParams = {}) {
+    return this.getTvSeason(tvId, seasonNumber, ['episode', episodeNumber, ...subPath], searchParams);
+  }
+
+  /**
+   * Get person by id as per https://developers.themoviedb.org/3/people/
+   * @param {string} personId
+   * @param {Array<string>} [subPath]
+   * @param {Object} [searchParams] - see query() for details
+   * @returns {Promise<Response>} - see query() for details
+   */
+  getPerson(personId, subPath = [],  searchParams = {}) {
+    return this.query('person', [personId, ...subPath], searchParams);
+  }
+
+  /**
+   * Commits a general purpose query
+   * Usage: query().then(data => {}).catch(rejection => {})
+   * @param {string} resource - like 'movie'
+   * @param {Array<string>} [pathParams] - e.g. [movieId, 'images']
+   * @param {Object} [searchParams] - see _makeGetUrl() for details
+   * @returns {Promise<Response>} - response body data or rejection
+   */
+  query(resource, pathParams = [], searchParams = {}) {
+    return fetch(this._makeGetUrl(resource, pathParams, searchParams))
+      .then(response => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw response;
+        }
+      })
+      .catch(rejection => {
+        throw rejection;
+      });
+  }
+
+  /**
+   * Gets image url of given size
+   * @param {string} filePath
+   * @param {number|string} size - call getImageryConfig() for available sizes per image type
+   * @returns {string}
+   */
+  getImageUrl(filePath, size) {
+    return this.apiConfig.imageBaseUrl + 'w' + size + filePath;
+  }
+
+  /**
+   * Gets imagery configuration from TMDB API server
+   * The response is being cached for re-use during same session
+   * @returns {Promise<any>} - cached data or query() result, see query() for details
+   */
+  getImageryConfig() {
+    return (this.imageConfig)
+      ? Promise.resolve(this.imageConfig)
+      : this.query('configuration').then(config => {
+        this.imageConfig = config;
+        return config;
+      });
+  };
+
+  /**
+   * Builds an URL for GET request from components, adding an api_key
+   * @param {string} resource - like 'movie'
+   * @param {Array<string>} [pathParams] - e.g. ['tv', tvId, 'season', seasonNumber]
+   * @param {Object} [searchParams] -
+   *        {key0:value0, key1:value1}, will transform into ?key0=value0&key1=value1,
+   *        keys and values are URL encoded for safety and consistency
+   * @returns {string}
+   * @private
+   */
+  _makeGetUrl(resource, pathParams = [], searchParams = {}) {
+    pathParams.unshift(resource);
+    searchParams = Object.assign(searchParams, {api_key: apiConfig.key});
```

Comment:
I think you can try to use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here

Reply:
Habit comes with practice... Thanks for pointing out at this. Fixed by a9de21f